posts/inside-rust/2023-08-21-clippy-perf-project.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Making Clippy blazingly fast by 2024"
+author: Alejandra González
+team: the Clippy team <https://www.rust-lang.org/governance/teams/dev-tools#Clippy%20team>
+description: This is the start of a series of posts talking about optimizations done on the Clippy codebase. 
+---
+
+Clippy's codebase is designed to be beginner-friendly so new contributors can come and learn,
+not optimized for performance. It was never a priority to make Clippy fast because *it never needed
+to be.* But time passed and more lints were added, now we have [over 650 lints][lint_list], each one of them adding a bit of performance overhead. In big projects this can take over a minute to check all those lints, so the Clippy team have decided to do something about it.
+
+> <span class="note"><svg class="note-icon" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path d="M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8Zm8-6.5a6.5 6.5 0 1 0 0 13 6.5 6.5 0 0 0 0-13ZM6.5 7.75A.75.75 0 0 1 7.25 7h1a.75.75 0 0 1 .75.75v2.75h.25a.75.75 0 0 1 0 1.5h-2a.75.75 0 0 1 0-1.5h.25v-2h-.25a.75.75 0 0 1-.75-.75ZM8 6a1 1 0 1 1 0-2 1 1 0 0 1 0 2Z"></path></svg> Full disclosure</span>
+>
+> While some members are being sponsored by Embark Studios to continue / improve these efforts, these optimizations are and will be always free and open source. No brands attached.
```

Comment:
```suggestion
> While Embark Studios is now sponsoring @blyxyas to continue / improve these efforts, these optimizations are and will be always free and open source. No brands attached.
```

You're owning this project, so be proud of it :) Feel free to put your name there instead/in addition to your GitHub tag.